README.md

```diff
@@ -139,16 +139,18 @@ And then we just close off the file.
 
 ### Configuring the control system
 
-The configuration file for the control system depends on the chosed control
+The configuration file for the control system depends on the chosen control
 system adapter. If no file is provided, it will default to expose everything to
 the control system.
 
 #### DOOCS
 
-The file for DOOCS has to be named
-`generic_chimeratk_server-DoocsVariableConfig.xml`. It might also be advisable,
-though not necessary, to provide a DOOCS server configuration file containing
-permissions and such. For details, refer to the [documentation of the DOOCS
+DOOCS requires a basic server configuration file named `doocs-generic-chimeratk-server01.conf` to set `SVR.RPC_NUMBER` 
+
+Additional configuration can be done through the DOOCS variable mapping file. It has to be named
+`generic_chimeratk_server-DoocsVariableConfig.xml`. If omitted, all variables are exported to DOOCS.
```

Comment:
```suggestion
`generic_chimeratk_server-DoocsVariableConfig.xml`. If omitted, all variables are exported to DOOCS, with the highest hierarchy level becoming the location name, and the other levels being converted into a dot-separated property name (i.e. /Module2/SubModule3/property7 becomes DOOCS property //Module2/SubModule3.property7)
```

Reply:
This should go into the DOOCS CSA, where this information is missing as well